content/chainguard/chainguard-enforce/iam-groups/verified-orgs.md

```diff
@@ -0,0 +1,77 @@
+---
+title : "Verified Organizations in Chainguard Enforce"
+linktitle: "Verified Organizations"
+lead: ""
+description: "An overview of how to verify your organization and the implications"
+type: "article"
+date: 2023-08-15T14:22:23-07:00
+lastmod: 2023-08-15T14:22:23-07:00
+draft: false
+tags: ["Enforce", "Product", "Conceptual"]
+images: []
+weight: 015
+---
+
+Resources on the Chainguard platform are organized in a hierarchical structure called [IAM Groups](/chainguard/chainguard-enforce/iam-groups/how-to-manage-iam-groups-in-chainguard-enforce/). A root-level IAM Group is called an _Organization_. Single customers or organizations typically only uses a single root-level _Organization_ to manage their
+Chainguard resources.
+
+Organizations can optionally be verified. Verification modifies some aspects of the Chainguard platform user experience to help large organizations guide their user base to the correct resources.
+
+## Verifing your Organization
+
+Verification is currently a manual process. To verify your organization, please contact your customer support contact. You can check if your organization is verified using [`chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/).
+
+```sh
+chainctl iam group ls -o json | jq
+```
+
+Verified organizations will have a field `verified: true` set.
+
+```json
+[
+  {
+    "id": "f5a2c73d75a8d7fe666ecb623c79a2b771d78765",
+    "name": "example.com",
+    "resourceLimits": {
+        "clusters": 3,
+        "idps": 1
+    },
+    "verified": true
+  }
+]
+```
+
+## Verified Organizations and Custom Identity providers
+
+If you've configured a [custom identity provider](/chainguard/chainguard-enforce/authentication/custom-idps/) and your organization is verified, you can select your identity provider by providing the name of your organization when authenicating.
+
+When authenticating with `chainctl`, the `--org-name` flag can be passed. Here, the command uses the example organization name `example.com`.
+
+```sh
+chainctl auth login --org-name example.com
+```
+
+As an alternative, you can set the organization name by editing the `chainctl` configuration file. You can do so with the following command.
+
+```sh
+chainctl config edit
+```
+
+This will open a text editor (nano, by default) where you can edit the local `chainctl` config. Add the following lines to this file.
+
+```yaml
+default:
+  org-name: example.com
+```
+
+Once set, the configured identity provider will be used automatically any time you run `chainctl auth login`.
+
+When authenticating via the Chainguard Console, your organization name is detected from your email address in most cases In cases where your organization name does not match your email domain, it can be input manually to select your custom identity provider.
```

Comment:
```suggestion
When authenticating via the Chainguard Console, your organization name is detected from your email address in most cases. If your organization name does not match your email domain, it can be input manually to select your custom identity provider.
```